app.py

```diff
@@ -28,6 +28,11 @@ def handle_bad_request(e):
     return jsonify(description=str(e.description)), 400
 
 
+@app.errorhandler(ValueError)
+def handle_value_error(e):
+    return jsonify(description=str(e)), 400
```

Comment:
Currently, we return a 404 for cases where a reference is invalid (ex. category ID not found for a criterion), but this changes all validation-related errors to a 400. I'm not sure what the norm is, so let me know if we'd like to change this!

Reply:
I think 400 is a good choice here!